From aa6cfea06fa2dc9a6f2acc719afc04b6f5b0b963 Mon Sep 17 00:00:00 2001
From: "Saravanan.K.S" <saravanan.kadambathursubramaniyam@windriver.com>
Date: Sat, 18 Nov 2023 10:52:28 +0000
Subject: [PATCH] Set default bootags to mount rootfs from eMMC device

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Saravanan.K.S <saravanan.kadambathursubramaniyam@windriver.com>
---
 include/configs/microchip_mpfs_icicle.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/microchip_mpfs_icicle.h b/include/configs/microchip_mpfs_icicle.h
index 5648980..81afe42 100644
--- a/include/configs/microchip_mpfs_icicle.h
+++ b/include/configs/microchip_mpfs_icicle.h
@@ -130,6 +130,7 @@
 #define CONFIG_EXTRA_ENV_SETTINGS \
 	"bootm_size=0x10000000\0" \
 	"scriptaddr=0x8e000000\0" \
+	"bootargs=earlycon=sbi root=/dev/mmcblk0p3 rootwait uio_pdrv_genirq.of_id=generic-uio\0" \
 	BOOTENV_DESIGN_OVERLAYS \
 	BOOTENV \
 
-- 
2.40.0

